eftcamb/03_abstract_EFTCAMB_cache.f90

```diff
@@ -226,6 +226,9 @@ end function Nu_pidotdot_Wrapper
         real(dl) :: EFTAT         !< the value of the tensor equation coefficient A. Refer to the Numerical Notes for the definition.
         real(dl) :: EFTBT         !< the value of the tensor equation coefficient B. Refer to the Numerical Notes for the definition.
         real(dl) :: EFTDT         !< the value of the tensor equation coefficient D. Refer to the Numerical Notes for the definition.
+        ! 12) Kinetic and Gradient quantities
```

Comment:
Make the comment slightly more informative like:
! 12) Kinetic and Gradient quantities for the stability check: